pkg/bootstrap/docker/up.go

```diff
@@ -84,11 +85,11 @@ var (
 	  %[1]s --host-data-dir=/mydata --use-existing-config
 
 	  # Use a different set of images
-	  %[1]s --image="registry.example.com/origin" --version="v1.1"`)
+	  %[1]s --image="registry.example.com/origin" --version="v1.1"
+
+	  # Use a different set of image streams
+	  %[1]s --image-streams="name:relative/path/to/image-streams.json"`)
 
-	imageStreamLocations = map[string]string{
-		"origin centos7 image streams": "examples/image-streams/image-streams-centos7.json",
-	}
```

Comment:
@gabemontero besides the e2e test which uses `cluster up` to stand up a cluster, no.
One of the challenges for cluster up is setting up tests that would run on Linux, Windows, and Mac. But we can definitely start with Linux.